api/graphql/reservations/reservation_serializers/refund_serializers.py

```diff
@@ -0,0 +1,63 @@
+import datetime
+from decimal import Decimal
+
+from django.utils.timezone import get_default_timezone
+
+from api.graphql.base_serializers import PrimaryKeySerializer
+from api.graphql.validation_errors import ValidationErrorCodes, ValidationErrorWithCode
+from merchants.models import OrderStatus, PaymentOrder
+from reservations.models import STATE_CHOICES, Reservation
+from reservations.tasks import refund_paid_reservation_task
+
+DEFAULT_TIMEZONE = get_default_timezone()
+
+
+class ReservationRefundSerializer(PrimaryKeySerializer):
+    def __init__(self, *args, **kwargs):
```

Comment:
Perhaps some leftover from testing(besthing)?

Reply:
I believe this is a doing of copypaste monster 😇
I'll remove these pointless override functions 🧹